layouts/partials/install.html

```diff
@@ -0,0 +1,29 @@
+<section id="install">
+    <div class="container content-center">
+  
+      <h2 class="install-title">Install</h2>
+      <hr class="install-underline">
+      <div class="grid-flex-container">
+        <div class="grid-flex-cell">
+            <ol class="list install-link-heading tc">
+                <li><h5><a class="link white" href="https://docs.ipfs.io/introduction/install/">1. Install IPFS on your machine</a></h5></li>
+                <li><h5>2. After installation, get started adding and working with files on IPFS.</h5></li>
+            </ol>
+            <article class="center mw5 mw6-ns hidden ba mv4 code-box">
+                <div class="pa3 bt tj code-box">
+                    <p class="code-comment"># create ipfs local config file</p>
+                    <p>ipfs init</p>
+                    <p class="code-comment"># start the ipfs deamon process</p>
+                    <p>ipfs daemon</p>
+                    <p class="code-comment"># show ipfs object data</p>
+                    <p>ipfs cat</p>
+                    <p class="code-comment"># download ipfs objects</p>
+                    <p>ipfs get</p>
+                </div>
+            </article>
+            <h5>Is the command line not your jam? Download <a href="https://github.com/ipfs-shipyard/ipfs-desktop">ipfs-desktop</a>!</h5>
+            </div>      
+        </div>
+    </section>
```

Comment:
There seems to be some indentation inconsistency here as the final </section> element is out of alignment with the opening <section> element.